agents/World.java

```diff
@@ -19,10 +22,19 @@
     private Map<String, Country> countryMap = new HashMap<>();
     private Map<String, City> cityMap = new HashMap<>();
 
-    public World(Path continentsPath, Path countriesPath, Path citiesPath) throws IOException {
+    private List<FemaleForename> femaleForenames = new ArrayList<>();
+    private List<MaleForename> maleForenames = new ArrayList<>();
+    private List<Surname> surnames = new ArrayList<>();
+
+    public static int ageOfAdulthood = 2;
```

Comment:
I'd prefer we use `public static final int AGE_OF_ADULTHOOD = 2;` as it the standard way to express constants in Java.

Reply:
Great, now I know the convention I've applied it throughout